### DISCUSSION TITLE
Variation Image Gallery For WooCommerce

### DISCUSSION THREAD

**Discussion by Goonlinejo** [inspireui/support#5531] [Working on Features]
Please add a feature that let us add a gallery image for each color variation.
![Screen Shot 2022-05-18 at 12 27 53 PM](https://user-images.githubusercontent.com/103630801/169006891-bdd18433-391b-48af-834b-1d40280c5d45.png)

**Comment by simonpham** [comment]
Hello @Goonlinejo,

Please update the app to our latest FluxStore version.
Since FluxStore 2.10.x, the gallery will automatically add images of all variants.

Thank you for your suggestion!
